pkg/server/sandbox_list.go

```diff
@@ -33,52 +27,30 @@ import (
 func (c *criContainerdService) ListPodSandbox(ctx context.Context, r *runtime.ListPodSandboxRequest) (*runtime.ListPodSandboxResponse, error) {
 	// List all sandboxes from store.
 	sandboxesInStore := c.sandboxStore.List()
-
-	response, err := c.client.TaskService().List(ctx, &tasks.ListTasksRequest{})
```

Comment:
whoot!!!

Reply:
Consider adding a todo for running a validate routine to make sure we have stayed in sync and didn't just miss an event.. I like this pattern of working off the push of events, but without some periodic validation there is a risk of an event failing to reach us, low but risk nonetheless? Or are all the events returning successful delivery to containerd?

Reply:
Good point. Will add a TODO for that.

We sometimes see Docker stay in weird state because it missed an event and not in sync with containerd.

Reply:
Done.